src/components/Avatar.js

```diff
@@ -2,8 +2,13 @@
 // returns an img element with className 'Avatar'
 // img element should have src and alt attributes
 
+import React from 'react';
+import { data } from '../store/data';
```

Comment:
Do not import data this way to lower level component, send necessary data to child components as props; see App.js data flow